tensorflow/python/distribute/distribute_lib.py

```diff
@@ -2895,7 +2901,7 @@ def update_non_slot(
 
   def _update_non_slot(self, colocate_with, fn, args, kwargs, group):
     raise NotImplementedError("must be implemented in descendants")
-
+  
```

Comment:
Please don't add trailing whitespace at the end of lines.
```suggestion

```

Reply:
Removed the trailing whitespace